features/steps/cantabularfeature.go

```diff
@@ -3,34 +3,66 @@ package steps
 import (
 	"context"
 	"encoding/json"
+	"errors"
 	"fmt"
+	"strings"
+	"testing"
 
 	"github.com/ONSdigital/dp-api-clients-go/v2/cantabular"
 	"github.com/ONSdigital/dp-api-clients-go/v2/cantabular/gql"
 	"github.com/ONSdigital/dp-cantabular-filter-flex-api/config"
 	"github.com/ONSdigital/dp-cantabular-filter-flex-api/features/mock"
 	"github.com/ONSdigital/dp-cantabular-filter-flex-api/service"
+	dphttp "github.com/ONSdigital/dp-net/v2/http"
 
 	"github.com/cucumber/godog"
 )
 
 type CantabularFeature struct {
-	*mock.CantabularClient
+	cantabularMock   *mock.CantabularClient
```

Comment:
I think cantabularClient was a clearer name to differentiate from cantabularServer